pkg/batch/jobs/cacher/segmentusercacher.go

```diff
@@ -85,6 +85,7 @@ func (c *segmentUserCacher) Run(ctx context.Context) error {
 			su := &ftproto.SegmentUsers{
 				SegmentId: seg.Id,
 				Users:     users,
+				UpdatedAt: seg.UpdatedAt,
```

Comment:
Here, we set the segment's `UpdatedAt`.
The same value will be returned to the SDK so it can be checked on the server when the SDK requests the segment users from the cache.